code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1949,7 +1949,7 @@ datum
 			fluid_g = 180
 			fluid_b = 240
 			transparency = 10
-			depletion_rate = 0.3
+			depletion_rate = 0.1
```

Comment:
```suggestion
			depletion_rate = 0.2
```
I get where you're coming from with this change, but 0.1 is just too low considering smoke powder exists.